### PR TITLE
Estimated delivery metadata

### PR DESCRIPTION
**Pull request by cdolm92**
## WHAT

Estimated delivery metadata now appears in Reward Cells and Backing Screen. 

**Reward Cell**
estimated-delivery-metadata

**Backing Screen**
screen shot 2017-02-02 at 5 48 02 pm

### CODE REVIEW

File: Library/ViewModels/RewardCellViewModel.swift [hunk 1]
```diff
@@ -148,15 +157,13 @@ RewardCellViewModelOutputs {
...
+       shouldCollapse(reward: reward, forProject: project)
     }
 
     self.allGoneHidden = projectAndReward
-      .map { project, reward in
-        reward.remaining != 0
-          || userIsBacking(reward: reward, inProject: project)
+        .map { project, reward in
+          reward.remaining != 0
+            || userIsBacking(reward: reward, inProject: project)
```
Comment (theginbin) [inline Library/ViewModels/RewardCellViewModel.swift]: i think you could set this spacing back to what it was?

File: Kickstarter-iOS/Views/Controllers/BackingViewController.swift [hunk 2]
```diff
@@ -15,6 +15,10 @@ internal final class BackingViewController: UIViewController {
   @IBOutlet fileprivate weak var backerSequenceLabel: UILabel!
   @IBOutlet fileprivate weak var backerShippingAmountLabel: UILabel!
   @IBOutlet fileprivate weak var backerShippingDescriptionLabel: UILabel!
+  @IBOutlet fileprivate weak var estimatedDeliveryDateLabel: UILabel!
+  @IBOutlet fileprivate weak var estimatedDeliveryLabel: UILabel!
+  @IBOutlet fileprivate weak var estimatedDeliverySeperatorView: UIView!
```
Comment (justinswart) [inline Kickstarter-iOS/Views/Controllers/BackingViewController.swift]: Spotted a couple misspellings of sep**a**rator.

File: Library/Tests/ViewModels/BackingViewModelTests.swift [hunk 3]
```diff
@@ -301,6 +303,22 @@ internal final class BackingViewModelTests: TestCase {
...
+    withEnvironment(apiService: MockService(fetchBackingResponse: .template
+      |> Backing.lens.reward .~ reward)) {
+        self.vm.inputs.configureWith(project: .template |> Project.lens.country .~ .US, backer: nil)
+
+        self.backerPledgeAmountAndDateAccessibilityLabel.assertValues([])
+
+        self.vm.inputs.viewDidLoad()
+
+        self.estimatedDeliveryDateLabelText.assertValues(["July 2016"],
+                                                                      "Emits estimated delivery date")
```
Comment (justinswart) [inline Library/Tests/ViewModels/BackingViewModelTests.swift]: I think the comment should be in line with the first argument:

```swift
        self.estimatedDeliveryDateLabelText.assertValues(["July 2016"],
                                                         "Emits estimated delivery date")
```

**Comment by cdolm92** [comment]
@kickstarter/native-squad g2g?

**Comment by mbrandonw** [comment]
Gonna merge this is cause realize the strings updated and so we'll get screenshot failures in other branches.

**Pull request merged**